odfe-cli/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+# Copied from https://opendistro.github.io/for-elasticsearch-docs/docs/install/docker/
+# removed node-2 since we don't need to create two node cluster for integration test
+version: '3'
+services:
+  odfe-test-node1:
+    image: amazon/opendistro-for-elasticsearch:1.11.0
```

Comment:
nit: recommend that the ODFE version isn't hardcoded here (if possible)

Reply:
How about this?
I will create an environment variable called ODFE_VERSION and set default as 1.11.0.
So if user set ODFE_VERSION = 1.10.0, it uses that value else default.

Reply:
I think I meant that we should try to hardcode the version in as few places as possible, so that when there is an upgrade, there are fewer places to change. 

That being said, I think it might be difficult to do this configuration outside this dock-compose file.

So, I like the idea of being able to configure with an env variable, but instead of hardcoding 1.11.0, could we instead just put in latest?

Reply:
Ack. Also i added environment variable for integration test to consume es credentials.